pkg/restapi/issuer/operation/operations.go

```diff
@@ -108,6 +108,10 @@ const (
 	credentialFulfillmentFormat    = "dif/credential-manifest/fulfillment@v1.0"
 	offerCredentialAttachMediaType = "application/json"
 	redirectStatusOK               = "OK"
+
+	// store prefixes
+	fulfilmentPrefix = "_fulfilment"
```

Comment:
Something I just learned: "fulfillment" and "fulfilment" (one L vs two Ls) are both acceptable spellings of the word! However, the spec uses the first spelling (two Ls), so we should be using that everywhere. Ref: https://identity.foundation/credential-manifest/#credential-fulfillment